src/Templates/Microsoft.CodeAnalysis.AnalyzerTemplate/MyProject.Analyzer.CodeFixes/MyProjectCodeFixProvider.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Rename;
+using Microsoft.CodeAnalysis.Text;
+
+namespace MyProject
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(MyprojectCodeFixProvider)), Shared]
+    public class MyProjectCodeFixProvider : CodeFixProvider
+    {
+        public sealed override ImmutableArray<string> FixableDiagnosticIds
+        {
+            get { return ImmutableArray.Create(MyProjectAnalyzer.DiagnosticId); }
+        }
```

Comment:
Don't this unnecessarily create a new array every time the property is accessed? I think it can be a one-time assignment like this.

```suggestion
        public sealed override ImmutableArray<string> FixableDiagnosticIds { get; } = ImmutableArray.Create(MyProjectAnalyzer.DiagnosticId);
```